src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/MobDropListener.java

```diff
@@ -48,14 +47,20 @@ public void onEntityKill(EntityDeathEvent e) {
     }
 
     private void addDrops(Player p, Set<ItemStack> customDrops, List<ItemStack> drops) {
+    	int random = ThreadLocalRandom.current().nextInt(100);
+    	
         for (ItemStack drop : customDrops) {
             if (Slimefun.hasUnlocked(p, drop, true)) {
-                if (circuitBoard != null && circuitBoard.isItem(drop) && !circuitBoard.isDroppedFromGolems()) {
+            	SlimefunItem sfi = SlimefunItem.getByItem(drop);
+            	if (sfi instanceof RandomMobDrop && ((RandomMobDrop)sfi).getDropChance() <= random) 
+            		continue;
+            	
+                if (sfi instanceof BasicCircuitBoard  && ((BasicCircuitBoard)sfi).isDroppedFromGolems()) 
                     continue;
```

Comment:
Why is this being continued? We want this to drop. You need to negate the isDropped...

Reply:
this issue still exists

Reply:
Fixed